setup.py

```diff
@@ -25,33 +25,25 @@
     long_description = fh.read()
 
 dependencies = [
-    # Dependency corrections from our requirements
-    "attrs==23.1.0",
-    "grpcio==1.53.2",
-    "lazy-object-proxy==1.9.0",
-    "marshmallow==3.19.0",
-    # Core dependencies
-    "fsspec>=2022.8.2",
-    "google-api-python-client==2.91.0",
-    "ibis-framework==5.1.0",
-    "impyla==0.18.0",
-    "SQLAlchemy==1.4.49",
-    "PyMySQL==1.1.1",
```

Comment:
PyMySQL is missing in the latest file version so our Cloud Build execution failed with the following error:

`Step #3 - "integration_mysql": E data_validation.exceptions.DataClientConnectionFailure: Connection Type "MySQL" could not connect: No module named 'pymysql'`

Reply:
Great, thank you! Started a new build execution, let's see how it goes 🍀 🤞